shared/modules/types.go

```diff
@@ -48,6 +48,11 @@ type PersistenceConfig interface {
 	GetBlockStorePath() string
 	GetTxIndexerPath() string
 	GetTreesStoreDir() string
+	GetMaxConnsCount() int32
+	GetMinConnsCount() int32
```

Comment:
Thoughts on making this unsigned?

EDIT: My guess is that you made them signed to correspond with https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool?

Reply:
Yep, I had to make the connection count type int32 to correspond to pgxpool connection count type